gandalf.js

```diff
@@ -1,120 +1,127 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!